DESCRIPTION

```diff
@@ -1,8 +1,8 @@
 Package: nanotime
 Type: Package
 Title: Nanosecond-Resolution Time Support for R
-Version: 0.3.7.2
-Date: 2023-03-12
+Version: 0.3.8
+Date: 2023-07-06
```

Comment:
I have a slight preference for 0.3.7.3 so that we can let this settle for a few days/weeks.  Not setting the next release number allows us to shimmy in another (similar) update before a possible next release.

Reply:
Now done.